raspberrypi-node.js

```diff
@@ -72,7 +72,7 @@ function initTasks(gulp, options) {
       nodejsParam = ' "' + config.iot_device_connection_string + '"';
     }
 
-    all.sshExecCmd('sudo' + ' ' + nodeCommand + ' ' + targetFolder + '/' + startFile + nodejsParam + ' && exit', { verbose: true }, cb);
+    all.sshExecCmd('sudo' + ' ' + nodeCommand + ' ' + targetFolder + '/' + startFile + nodejsParam + ' && exit', { verbose: args.verbose }, cb);
   });
```

Comment:
I thought we have overridden this on purpose. So will you pass --verbose from the sample code I assume?